app/src/features/projects/components/ProjectPermitForm.tsx

```diff
@@ -87,6 +87,16 @@ const ProjectPermitForm: React.FC = () => {
 
   const { values, handleChange, getFieldMeta, errors } = useFormikContext<IProjectPermitForm>();
 
+  const permitTypes = [
+    'Forestry License to Cut',
+    'Heritage Inspection Permit',
+    'License of Occupation',
+    'Occupant License to Cut under the Forest Act',
+    'Park Use Permit',
+    'Road Use Permit',
+    'Special Use Permit'
+  ].sort();
```

Comment:
nit: since this is a static list, we may as well just create this array in sorted order to begin with, and then drop the `.sort()` call.

Reply:
Yea I just suggested it because then we dont have to always keep an eye on it even in future when list changes if the list is sorted or not even if it is static.

Reply:
oh gotcha, yeah fair point.  I wasn't thinking of that when I commented.  I'm not sure what the best option is, I guess I'd still lean towards having it pre-sorted? Mostly because it feels like a waste (albeit a super minor one) to sort it every time.  Not sure though.

Reply:
I'd probably just lean on the rule of "dont prematurely optimize" in this case.  Keep it simple until it becomes an issue.

Reply:
Perfect yea sounds great 👍🏼